package.json

```diff
@@ -63,7 +63,6 @@
     "sinon-chai": "^3.5.0",
     "ts-loader": "^7.0.5",
     "ts-mock-imports": "^1.3.0",
-    "ts-node": "^8.10.2",
```

Comment:
Why do we remove it since we are using it in scripts?

Reply:
In the readme, we actually encourage them to run with globally those package installed so that they can manage them, have them reuse across project, in the shell, and it's quite heavy.
If we are a typescript SDK, and expect people to work with typescript (because we use that language), we expect them to have typescript & ts-node existing on their machine already. 

We could specify it, I've saw `nodemon` sometimes in package.json, but yeah, that is the reason why I removed it. 
What should I do ?

Reply:
@antouhou What do you think about it?

Reply:
I would say that if we remove `ts-node`, then we need to change the script to something like `tsc ... && node ...js`